packages/hint/docs/contributor-guide/how-to/hint.md

```diff
@@ -94,15 +94,29 @@ with the website and report errors.
 To report an error, the hint has to do the following:
 
 ```ts
-await context.report(resource, element, message);
+await context.report(resource, message, { element: element });
 ```
 
 * `context.report()` is an asynchronous method, you should always `await`.
 * `resource` is the URL of what is being analyzed (HTML, JS, CSS, manifest,
   etc.)
-* `element` is the `IAsyncHTMLElement` that triggered the problem. Not always
-  necessary. In the case of an image, script, style, it will be an `img`,
-  `script`, `link`, etc.
+* `message` is the text to show to the user about the problem.
+* `element` is an optional `IAsyncHTMLElement` where the issue was found
```

Comment:
I think this is a bit confusing. The third parameter is the `options` object that can have an `element` property, right? I think we should make it clearer. Right now this splits the description between `element` and `options`. What about something like:

```ts
await context.report(resource, message, options);
```
* `context.report()` is an asynchronous method, you should always `await`.
* `resource` is the URL of what is being analyzed (HTML, JS, CSS, manifest,
  etc.)
* `message` is the text to show to the user about the problem.
* options is an (optional) object that can contain the following:
  * `element` is an optional `IAsyncHTMLElement` where the issue was found ...
  * `codeSnippet` is a string of source code to display (defaults to the
  `outerHTML` of `element`).
  * ...